plugins/modules/identity/ipa/ipa_pwpolicy.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2020, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ipa_pwpolicy
+author: Adralioh (@adralioh)
+short_description: Manage FreeIPA password policies
+description:
+- Add, modify, and delete password policy within IPA server.
+version_added: 1.3.0
+options:
+    cn:
+        description:
+        - Name of the group that the policy will apply to.
+        - If omitted, the global policy will be used.
+        aliases: ["group", "name"]
```

Comment:
```suggestion
        aliases: ["group", "name"]
```
i would avoid aliases which is ok to add sometimes (but not initially), e.g. when a typo or something. So i'd choose one best option for every parameter.